app/code/Magento/Pwa/Test/Mftf/ActionGroup/VeniaCartPageActionGroup.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="GoToCartPage">
+        <amOnPage url="{{Cart.url}}" stepKey="goToCartPage"/>
+        <wait time="5" stepKey="temporaryStaticWait1" />
+    </actionGroup>
+
+    <actionGroup name="RemoveItemFromTheCartPage">
+        <arguments>
+            <argument name="product" defaultValue="ValeriaTwoLayerTank"/>
+        </arguments>
+        <click selector="{{CartPageKebabSection.kebabMenu(product.name)}}" stepKey="clickOnKebabMenu"/>
+        <wait time="3" stepKey="wait123"/>
+        <waitForPwaElementVisible selector="{{CartPageKebabSection.removeItem}}" stepKey="waitForKebabMenuToExpand"/>
+        <click selector="{{CartPageKebabSection.removeItem}}" stepKey="clickOnRemoveItem"/>
+        <wait time="10" stepKey="wait1234"/>
```

Comment:
Do all steps need a `stepKey`? Do they have to be unique?

Reply:
Yes, unique step keys needed else you will see duplicate stepkey error when you run test.